indico/modules/rb_new/client/js/modules/calendar/actions.js

```diff
@@ -35,6 +37,12 @@ export const FETCH_SUCCESS = 'calendar/FETCH_SUCCESS';
 export const FETCH_ERROR = 'calendar/FETCH_ERROR';
```

Comment:
`FETCH_CALENDAR_...` and then `FETCH_ACTIVE_BOOKINGS_...` for your new ones to be a bit more consistent